src/smp.rs

```diff
@@ -30,10 +34,11 @@ pub struct PerCpu {
     page_table: HypPageTable,
     umode_task: Once<RefCell<UmodeTask>>,
     online: Once<bool>,
+    stack_top: u64,
 }
 
 /// The number of pages we allocate per CPU: the CPU's stack + it's `PerCpu` structure.
-const PER_CPU_PAGES: u64 = 8;
+const PER_CPU_PAGES: u64 = 0x80;
```

Comment:
512k per CPU?!

Reply:
Yes. Our stack usage isn't great. Mapping takes quite a lot. And when we run in Debug Mode we're essentially all over the place when it come to stack usage.

I am due to restore my previous stack analysis once this is over.